codice/ddf
# DDF-1265: Migrate ddf.docs into ddf.distribution

I moved the ddf docs project into the ddf/distribution directory so that the ddf documentation will take on the version of the distribution. Previously, the documentation was taking on the version of ddf-parent.

To verify that this was done correctly:
1.) Verify that everything builds
2.) Verify that the documentation is generated correctly with the correct version (2.8.0-SNAPSHOT). To do this:
* Add the following tag to the ddf-parent release profile:

```
<activation>
    <file>
        <exists>${basedir}/src/assembly/bin.xml</exists>
    </file>
</activation>
```

* Build ddf-parent with `mvn clean install`
* Build ddf with `mvn clean install`


Furthermore, I ran into some dirty words in the documentation when I was committing, so I removed those (all the serious dirty word occurrences). There still are occurrences of the following: mil, gov, nga


[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/codice/ddf/53)
